pages/linux/ffsend.md

```diff
@@ -0,0 +1,23 @@
+# ffsend
+
+> Easily and securely share files from command line.
+
```

Comment:
Perhaps we could add a line here like this:

```markdown
> Homepage: <https://gitlab.com/timvisee/ffsend>.
```

Reply:
added